spec/services/importers/party_data_spec.rb

```diff
@@ -4,7 +4,7 @@
   describe '#perform(oscn_id)' do
     it 'saves aliases' do
       path = 'spec/fixtures/party_example.html'
-      html = File.open(path).read
+      html = File.read(path)
```

Comment:
did you mean to have this?

Reply:
I think this is a rubocop thing. it seems to prefer  `File.open(path).read` over `File.read(path)` If i remember correctly